.drone.yml

```diff
@@ -15,6 +15,50 @@ steps:
   - npm install typedoc
   - npx typedoc src/plot-data.ts --out doc/build/typescript/
 
+- name: install, run scripts and build doc
+  image: python:3.8
+  commands:
+  - git fetch --tags
+  - python setup.py install
+#  - python scripts/dessia_object.py
+  - pip install coverage #sphinx sphinx-rtd-theme coverage nbformat nbconvert
```

Comment:
Reminder that we probably need to remove commented elements

Reply:
Done

Reply:
But the sphinx one which i like here for the moment